apps/sentry-client-desktop/src/features/keys/modals/buy-keys/BuyKeysFlow.tsx

```diff
@@ -5,7 +5,7 @@ import {BuyFlowBanner} from "@/features/keys/modals/buy-keys/BuyFlowBanner";
 import { RedSentryIcon } from "@sentry/ui/src/rebrand/icons/IconsComponents";
 
 export function BuyKeysFlow() {
-	const [displayQuantity, setDisplayQuantity] = useState<number>(1);
+	const [displayQuantity, setDisplayQuantity] = useState<number>(10);
```

Comment:
This was out of scope for the ticket I think, thanks for doing it, make sure to mention in the ticket